docs/src/developing/programming-model/runtime.md

```diff
@@ -54,8 +54,9 @@ to.
 As the transaction is processed compute units are consumed by its
 instruction's programs performing operations such as executing BPF instructions,
 calling syscalls, etc... When the transaction consumes its entire budget, or
-exceeds a bound such as attempting a call stack that is too deep, the runtime
-halts the transaction processing and returns an error.
+exceeds a bound such as attempting a call stack that is too deep, or loaded 
+account data exceeds limit, the runtime halts the transaction processing and
+returns an error.
```

Comment:
Planning to have `accounts::load_transaction` to honor `ComputeBudgetInstruction::set_accounts_data_size_limit`, assume runtime can do the same, would that be the case? @Lichtso @jackcmay